cpp/include/cudf/io/types.hpp

```diff
@@ -99,6 +99,26 @@ enum statistics_freq {
   STATISTICS_COLUMN   = 3,  ///< Full column and offset indices. Implies STATISTICS_ROWGROUP
 };
 
+/**
+ * @brief Valid encodings for use with `column_in_metadata::set_encoding()`
+ */
+enum class column_encoding {
+  // common encodings
+  NOT_SET = -1,  ///< No encoding has been requested
```

Comment:
`NOT_SET` is confusing. It implicitly means the value is default-initialized to something, not to "no encoding". So maybe we should call it `NO_ENCODING`?

Reply:
It means `set_encoding()` was not called. I guess the effect is the default encoding for the data type will be used, so maybe `DEFAULT` or `USE_DEFAULT`?

Reply:
`UES_DEFAULT` sounds better to me.